run/test/specs/utils/chrome_first_time_open.ts

```diff
@@ -0,0 +1,14 @@
+import { DeviceWrapper } from "../../../types/DeviceWrapper";
+import { ChromeNotificationsNegativeButton, ChromeUseWithoutAnAccount } from "../locators/browsers";
+
+// First time open of Chrome triggers an account check and a notifications modal 
+export async function isChromeFirstTimeOpen(device: DeviceWrapper) {
+const chromeUseWithoutAnAccount = await device.doesElementExist(new ChromeUseWithoutAnAccount(device))
```

Comment:
I would add a maxWait here of 500ms, it doesn't need to wait that long

Reply:
reduced to 500 in the locator file